packages/dashboard/src/pages/Personas/components/UploadAvatarDialog/index.tsx

```diff
@@ -0,0 +1,84 @@
+import { MaskDialog } from '@masknet/theme'
+import { Box, Button, DialogContent, Slider } from '@material-ui/core'
+import { memo, useCallback, useState } from 'react'
+import AvatarEditor from 'react-avatar-editor'
+import { useStateList } from 'react-use'
+import FileUpload from '../../../../components/FileUpload'
+import { useDashboardI18N } from '../../../../locales'
+import { updatePersonaAvatar } from '../../api'
+
+interface UploadAvatarDialogProps {
+    open: boolean
+    onClose(): void
+}
+
+const uploadSteps = ['upload', 'pick']
```

Comment:
> I suggest using enum instead of Step.

`const { state: step, next, setStateAt } = useStateList(uploadSteps)`

The parameters of this useStateList are arrays.